### ISSUE TITLE
Suggests for auto prompt

### ISSUE THREAD

**Issue by maxe-hsieh** [CG8516/PvZA11y#3]
Hello, first of all, thank you very much for creating this mod with great care, allowing visually impaired players to play this game smoothly. However, I have some feedback that I would like to provide for your reference.
1. I hope that you can add more accurate prompts when zombies appear, using sound height to represent their positions. For example, from left to right, we can use letters A-I, and from top to bottom, we can use numbers 1-6. When a zombie appears at 1i, the sound should be the highest, slightly lower for 2i, and so on.
2. I hope that when moving up and down, we can directly have the number of zombies in each row read out from left to right, and even the types of zombies, etc. In other words, every time the focus is moved, it automatically performs the function of pressing F1 for us. This way, players can quickly understand the overall situation of the game.

**Comment by CG8516** [comment]
No problem at all, thanks for the feedback!  

1. I'll add that as an optional feature, but it may be overwhelming during waves.

2. The latest update (beta 1.8) now has this as an accessibility option.

**Comment by maxe-hsieh** [comment]
I apologize, perhaps I didn't express the issue clearly.
Currently, when each zombie appears at the moment between I1 and I5, we are unaware. Certainly, we can use F1 to manually check the number of zombies in each row, but this leads to the need to check each row using F1 every few seconds, which is quite cumbersome.
Therefore, we propose using the beep sound or any means to automatically provide a notification when any zombie appears in the moment between I1 and I5.
Testing method: When we take no operation and are still able to determine the position of each zombie appearing between I1 and I5, then the test is considered successful.

**Comment by CG8516** [comment]
I understand what you mean, my message probably wasn't clear either, sorry.  

I've implemented the second thing you suggested in your post, where the scanner / F1 functionality will now happen when moving up or down on the board.  

And I will add your other suggestion (an automatic pitched audio cue for zombies entering from any row), in the next update.

**Comment by CG8516** [comment]
Added in beta 1.9

**Comment by c469591** [comment]
Hello, I hope that the function of F2 can be automatically executed when I use the up, down, left, and right keys to read aloud the plant in the current cell. This is especially important when there's water because I alternate planting different plants and might not remember where I planted leaves last time. Therefore, repeatedly pressing F2 makes my operation busier. Thank you.

**Comment by CG8516** [comment]
That feature is already included. It's the option called "Say Object on move" in the accessibility settings.

If you press Info1 on each accessibility setting, it will provide a longer description of what each one does.

I definitely need to rename that setting to something a bit more clear though, because I can totally understand why somebody might not realise that it includes plants.

**Comment by c469591** [comment]
By the way, the current cell's terrain; as well as zombies might also need to be announced,
terrain such as grass, potted plants, water,
zombies such as regular ones, pole-vaulting and so on.
The overall priority could be as follows.
Plants, Zombies, Terrain
For instance, when there are plants and zombies.
Peashooter; Regular Zombie, swimming pool
Only when there are zombies
Ordinary zombies, swimming pool
Only when there are plants.
Pea Shooter, Swimming Pool
When there are no zombie plants
Swimming Pool

Additionally, the reading order can be directly followed as I've described above, without needing to adhere to the typical English expression.
Just don't do it like this.
Peashooter in the Pond
Because using the translation function might affect the order of reading aloud, what we need to know first is about plants, not terrain.
The above are my thoughts. My English isn't very good, and I've used a translator, so I hope it's understandable. Thank you.
